core/src/edu/asu/c3simulator/simulation/business/ProductionInformation.java

```diff
@@ -0,0 +1,60 @@
+package edu.asu.c3simulator.simulation.business;
+
+/**
+ * Structure to represent how, and in what quantities a {@link Product} is being produced.
+ * 
+ * @author Moore, Zachary
+ *
+ */
+public class ProductionInformation
+{
+	/** Product for which this information applies */
+	private Product target;
+	
+	/** Number of units of {@link #target} being produced by this company each month */
+	private int monthlyProduction;
+	
+	/** Amount spent on materials per unit produced */
+	private float materialBudgetPerUnit;
+	
+	public int getMonthlyProduction()
+	{
+		return monthlyProduction;
+	}
+	
+	public void increaseMonthlyProduction(int amount)
+	{
+		monthlyProduction += amount;
+	}
+	
+	public void setMonthlyProduction(int monthlyProduction)
+	{
+		this.monthlyProduction = monthlyProduction;
+	}
+	
+	public float getMaterialBudgetPerUnit()
+	{
+		return materialBudgetPerUnit;
+	}
+	
+	public void increaseMaterialBudgetPerUnit(int amount)
+	{
+		materialBudgetPerUnit += amount;
+	}
+	
+	public void setMaterialBudgetPerUnit(float materialBudgetPerUnit)
+	{
+		this.materialBudgetPerUnit = materialBudgetPerUnit;
+	}
+	
+	public Product getTarget()
+	{
+		return target;
+	}
+	
+	public float getTotalMonthlyMaterialExpense()
```

Comment:
I had to think about this one because I was not 100% sure if you could multiply an int (monthlyProduction) times a float (materialBudgetPerUnit) but, I'm sure the result of those two would be a float, which is what you have.